### PR TITLE
[Feature] Add SVTP to dataset preparer

### PR DESCRIPTION
**Pull request by xinke-wang**
This PR relies on the updated `ICDARTxtTextRecogAnnParser` in #1529

### CODE REVIEW

**Comment by xinke-wang** [comment]
Since the `svtp` is a variant of the original `svt` dataset, the official web of `svtp` given in the paper no longer holds. Most of the current versions are held on GitHub and uploaded by a third party. It might be better to upload the dataset on OpenMMLab's server and replace the tmp link here later.

File: dataset_zoo/svtp/textrecog.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+data_root = 'data/svtp'
+cache_path = 'data/cache'
+
+data_obtainer = dict(
+    type='NaiveDataObtainer',
+    cache_path=cache_path,
+    data_root=data_root,
+    files=[
+        dict(
+            url='https://cloudstor.aarnet.edu.au/plus/s/U7Tw4KnqkjSBNpE/'
```
Comment (Harold-lkk) [inline dataset_zoo/svtp/textrecog.py:10]: add TODO

**Review by Harold-lkk — APPROVED** [review]

File: dataset_zoo/svtp/textrecog.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+cache_path = 'data/cache'
+
+data_obtainer = dict(
+    type='NaiveDataObtainer',
+    cache_path=cache_path,
+    data_root=data_root,
+    files=[
+        dict(
+            # TODO: Upload the file to OpenMMLab's server and replace the link.
+            url='https://cloudstor.aarnet.edu.au/plus/s/U7Tw4KnqkjSBNpE/'
```
Comment (gaotongxiao) [inline dataset_zoo/svtp/textrecog.py:11]: https://download.openmmlab.com/mmocr/data/svtp.zip

File: dataset_zoo/svtp/metafile.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+Name: 'Street View Text Perspective (SVT-P)'
+Paper:
+  Title: https://openaccess.thecvf.com/content_iccv_2013/papers/Phan_Recognizing_Text_with_2013_ICCV_paper.pdf
```
Comment (gaotongxiao) [inline dataset_zoo/svtp/metafile.yml:3]: Is the title correct?

Comment (xinke-wang) [reply dataset_zoo/svtp/metafile.yml]: fixed

**Review by gaotongxiao — APPROVED** [review]

**Pull request merged**